add advisory for cell-project (#1391)
rustsec/advisory-db: crates/cell_project/RUSTSEC-0000-0000.md

```toml
[advisory]
id = "RUSTSEC-0000-0000"
package = "cell-project"
date = "2020-08-27"
url = "https://github.com/RustyYato/cell-project/issues/3"
categories = ["memory-corruption"]
keywords = ["cell", "subtype", "variance"]

> informational = unsound

Reviewer: ```suggestion
informational = "unsound"
```

> 

Reviewer: ```suggestion
references = ["https://github.com/RustyYato/cell-project/issues/3", "https://github.com/RustyYato/cell-project/issues/4"]

```

    |  _____________________________^
15  | |                 x: None,
16  | |             }).x.as_ref().is_none();
    | |______________^
    = note: inside `std::ptr::drop_in_place::<Foo> - shim(Some(Foo))` at $RUST_STD_PATH/src/rust/library/core/src/ptr/mod.rs:486:1
    = note: inside `std::ptr::drop_in_place::<std::cell::UnsafeCell<Foo>> - shim(Some(std::cell::UnsafeCell<Foo>))` at $RUST_STD_PATH/src/rust/library/core/src/ptr/mod.rs:486:1
    = note: inside `std::ptr::drop_in_place::<std::cell::Cell<Foo>> - shim(Some(std::cell::Cell<Foo>))` at $RUST_STD_PATH/src/rust/library/core/src/ptr/mod.rs:486:1
note: inside `main` at src/main.rs:32:1
   --> src/main.rs:32:1
    |
32  | }
    | ^

note: some details are omitted, run with `MIRIFLAGS=-Zmiri-backtrace=full` for a verbose backtrace

error: aborting due to previous error
```

## Affected Versions

All versions of the cell-project crate before `0.1.4` are affected.

## Mitigation

This was fixed in https://github.com/RustyYato/cell-project/pull/4, and released as version `0.1.4`.
So just updating to the latest version will include the fix (which may result in a compile error on unsound usage).

## Acknowledgements

> This was discovered and fixed by @Soni here: https://github.com/RustyYato/cell-project/issues/3 and https://github.com/RustyYato/cell-project/issues/4 respectively.

Reviewer: ```suggestion
This was discovered and fixed by @SoniEx2 here in cell-project: [Issues/3] and [Issues/4] respectively.

[Issues/3]: https://github.com/RustyYato/cell-project/issues/3
[Issues/4]: https://github.com/RustyYato/cell-project/issues/4
```
